.github/workflows/label.yml

```diff
@@ -1,6 +1,6 @@
 name: "Pull Request Labeler"
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
The `if` conditions (see line 19) requires this to be run on `pull_request` events so that we can properly detect the head branch of the fork itself.

Reply:
These can be changed once we agree on whether we should do this first or not.